docs/catalog/cloud/use-compiled-languages.md

```diff
@@ -0,0 +1,44 @@
+---
+version: 1.0
+submitted_by: markus-ntt-seidl
+published_date: TBD
+category: cloud
+tags: 
+ - cloud
+ - role:cloud-engineer
+ - size:small
+---
+
+# Use compiled languages
+
+## Description
+
+Interpreted languages need to be parsed, compiled and executed when the application starts or a workload arrives. This tends to be more energy heavy then when a compiled language is used. The compilation is then only done once, saving on resources.
+
+## Solution
+
+Use compiled languages whenever possible or compile interpreted languages.
```

Comment:
Any example or two here of a compiled language (Java, Go, Rust) will help with setting context and help connect this pattern to search engines/crawlers for popular languages